app/models/applicant.rb

```diff
@@ -14,4 +14,17 @@ class Applicant < ApplicationRecord
   def validate_date_of_birth
     errors.add(:date_of_birth, 'is not valid') if date_of_birth.present? && (date_of_birth < Date.new(1900, 0o1, 0o1) || date_of_birth > Date.today)
   end
+
+  def normalise_national_insurance_number
```

Comment:
**[not a blocker]** I would normally normalise the NINo on initialisation of the object if the value was present. It took me a little bit to realise we're effectively changing the value during the validation itself.